zcash_client_backend/src/decrypt.rs

```diff
@@ -159,7 +158,7 @@ pub fn decrypt_transaction<'a, P: consensus::Parameters, AccountId: Copy>(
         .collect();
 
     #[cfg(feature = "orchard")]
-    let orchard_bundle = tx.orchard_bundle();
+    let orchard_bundle = tx.orchard_bundle().map(|bundle| bundle.vanilla_bundle());
```

Comment:
has the same meaning:
`let orchard_bundle = tx.orchard_bundle().unwrap().vanilla_bundle();`